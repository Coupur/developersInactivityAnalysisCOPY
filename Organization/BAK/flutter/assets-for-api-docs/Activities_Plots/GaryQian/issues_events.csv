id,date,event,creator_login
2494304920,2019-07-18 23:28:56,review_requested,GaryQian
2494304922,2019-07-18 23:28:56,assigned,GaryQian
2494403704,2019-07-19 00:36:49,referenced,GaryQian
2494403705,2019-07-19 00:36:49,merged,GaryQian
2494403708,2019-07-19 00:36:49,closed,GaryQian
2493902789,2019-07-18 20:15:50,review_requested,GaryQian
2494039980,2019-07-18 21:11:42,referenced,GaryQian
2494039981,2019-07-18 21:11:42,merged,GaryQian
2494039983,2019-07-18 21:11:42,closed,GaryQian
2467727562,2019-07-08 23:47:22,referenced,GaryQian
2467727563,2019-07-08 23:47:22,merged,GaryQian
2467727567,2019-07-08 23:47:23,closed,GaryQian
2467570933,2019-07-08 22:06:42,referenced,GaryQian
2467570935,2019-07-08 22:06:42,merged,GaryQian
2467570942,2019-07-08 22:06:42,closed,GaryQian
2425591184,2019-06-19 21:34:03,review_requested,GaryQian
2425591185,2019-06-19 21:34:03,review_requested,GaryQian
2425676739,2019-06-19 22:18:57,referenced,GaryQian
2425676741,2019-06-19 22:18:57,merged,GaryQian
2425676749,2019-06-19 22:18:57,closed,GaryQian
2422624468,2019-06-19 00:59:44,assigned,GaryQian
2425680894,2019-06-19 22:21:21,closed,GaryQian
