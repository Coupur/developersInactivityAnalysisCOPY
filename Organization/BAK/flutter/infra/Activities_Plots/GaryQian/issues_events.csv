id,date,event,creator_login
2159189236,2019-02-22 23:15:21,mentioned,GaryQian
2159189237,2019-02-22 23:15:21,subscribed,GaryQian
