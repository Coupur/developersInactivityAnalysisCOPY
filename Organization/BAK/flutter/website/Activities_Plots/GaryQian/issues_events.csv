id,date,event,creator_login
2364101126,2019-05-24 00:41:39,review_requested,GaryQian
2372839408,2019-05-28 19:07:42,referenced,GaryQian
2372839414,2019-05-28 19:07:42,merged,GaryQian
2372839432,2019-05-28 19:07:42,closed,GaryQian
2289188713,2019-04-19 23:53:52,closed,GaryQian
1885381782,2018-10-04 18:25:15,review_requested,GaryQian
1891008432,2018-10-08 18:26:53,review_requested,GaryQian
