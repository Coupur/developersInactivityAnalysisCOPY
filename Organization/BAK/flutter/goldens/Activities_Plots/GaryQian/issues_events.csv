id,date,event,creator_login
2357314961,2019-05-21 20:08:22,mentioned,GaryQian
2357314962,2019-05-21 20:08:22,subscribed,GaryQian
1831682936,2018-09-06 23:29:49,head_ref_deleted,GaryQian
